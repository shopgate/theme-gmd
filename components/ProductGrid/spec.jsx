import React from 'react';
import { shallow } from 'enzyme';
import ProductGrid from '../ProductGrid';

describe('<ProductGrid />', () => {
  it('should', () => {
    const wrapper = shallow(<ProductGrid />);

    expect(wrapper).toMatchSnapshot();
  });
});
