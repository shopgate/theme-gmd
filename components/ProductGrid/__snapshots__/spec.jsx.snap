// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductGrid /> should 1`] = `
<InfiniteContainer
  initialLimit={6}
  items={null}
  iterator={[Function]}
  limit={30}
  loader={[Function]}
  loadingIndicator={<LoadingIndicator />}
  preloadMultiplier={2}
  totalItems={null}
  wrapper={[Function]}
/>
`;
